Yoast/javascript
# Create Select components

## Summary
This PR adds two new components: `Select` and `MultiSelect`. `Select` is a wrapper around the default HTML select while `MultiSelect` uses the `select2` package to create a select.


This PR can be summarized in the following changelog entry:
* [yoast-components] Adds a `Select` and `MultiSelect` component.

## Relevant technical choices:
* I have tried to create an abstraction for overlap of code between the `MultiSelect` and the `Select` components. However, I have been unable to create a suitable one without any errors. If someone has a better idea, feel free to try it.

## Test instructions

This PR can be tested by following these steps:
* 🔥-up the example components app and checkout the `Select` and `MultiSelect`. Check that they behave as expected.

## Quality assurance

Fixes #560
